WooCommerce/Classes/POS/ViewModels/PointOfSaleDashboardViewModel.swift

```diff
@@ -35,17 +52,26 @@ final class PointOfSaleDashboardViewModel: ObservableObject {
 
     @Published private(set) var orderStage: OrderStage = .building
 
+    @Published private var order: PointOfSaleOrder?
```

Comment:
Should this property be Optional if we always plan to have an ongoing `PointOfSaleOrder` at this point? What about holding some sort of _"empty"_ PointOfSaleOrder? This could remove all the optionality handling around order totals, as well as reset it as nil when we create a new transaction, and as we build the order we just create new copies 🤔